src/javascripts/ng-admin/Crud/filter/maFilterController.js

```diff
@@ -18,9 +18,9 @@ define(function () {
         this.$state = $state;
         this.$stateParams = $stateParams;
         this.$filter = $filter;
-        this.values = this.$stateParams.search || {};
+        this.$scope.values = this.$stateParams.search || {};
```

Comment:
Allows filterWidget definitions to look like

```js
function getFilterWidget() {
    return '<ma-input-field field="::field" value="values[field.name()]"></ma-input-field>';
}
```

Instead of:

```
function getFilterWidget() {
    return '<ma-input-field field="::field" value="filterCtrl.values[field.name()]"></ma-input-field>';
}
```